source/faq/faq.rst

```diff
@@ -123,115 +119,139 @@ Devices
 What are *devices* for dojot?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In dojot, a device is a digital representation of an actual device or gateway with one 
-or more sensors or of a virtual one with sensors/attributes inferred from other devices.
+In dojot, a device is a digital representation of an actual device or gateway
+with one or more sensors or of a virtual one with sensors/attributes inferred
+from other devices.
 
-Consider, for instance, an actual device with temperature and humidity sensors; it can be 
-represented into dojot as a device with two attributes (one for each sensor). We call this 
-kind of device as *regular device* or by its communication protocol, for instance, 
-*MQTT device* or *CoAP device*.
+Consider, for instance, an actual device with temperature and humidity sensors;
+it can be represented into dojot as a device with two attributes (one for each
+sensor). We call this kind of device as *regular device* or by its
+communication protocol, for instance, *MQTT device* or *CoAP device*.
 
-We can also create devices which don’t directly correspond to their actual ones, for instance, 
-we can create one with higher level of information of temperature (*is becoming hotter* or 
-*is becoming colder*) whose values are inferred from temperature sensors of other devices. 
-This kind of device is called *virtual device*.
+We can also create devices which don’t directly correspond to their actual
+ones, for instance, we can create one with higher level of information of
+temperature (*is becoming hotter* or *is becoming colder*) whose values are
+inferred from temperature sensors of other devices. This kind of device is
+called *virtual device*.
 
 What is the relationship between this *device* and my actual device?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It is simple as it seems: the *regular device* for dojot is a mirror (digital twin) of 
-your actual device. You can choose which attributes are available for applications and 
-other components by adding each one of them at the device creation interface. If you don't 
-want some attributes to be available to applications or other elements, 
-just don't add them in dojot.
+It is simple as it seems: the *regular device* for dojot is a mirror (digital
+twin) of your actual device. You can choose which attributes are available for
+applications and other components by adding each one of them at the device
+creation interface. If you don't want some attributes to be available to
+applications or other elements, just don't add them in dojot.
```

Comment:
> If you don't want some attributes to be available to applications or other elements, just don't add them in dojot.

This sounds quite awful to me. We probably should have a way of limiting what is accessible by applications at some point without having to worry with not modeling and storing that data on dojot for our own "different" use.

Reply:
We could discuss this for the next version. I'll remove the last sentence, which seems to me too rude.

Reply:
Indeed it is.